packages/metrics/src/index.ts

```diff
@@ -7,18 +7,22 @@ export type IgnoreError = string | RegExp;
 export type Settings = {
   context?: { [key: string]: unknown };
   ignoreErrors?: IgnoreError[];
+  sentryDSN?: string;
   sentryKey?: string;
   sentryProject?: string;
   userID?: number | null;
+  onSentryScope?: ((scope: Scope) => void) | null;
```

Comment:
not super clear what the intention of this is.

Reply:
Allows consumers to hook into sentry since it's abstracted away.

```
Metrics.initialize({
  onSentryScope(scope => {
    scope.addTag('custom', 'consumer specific');
  }),
});
```

Reply:
Any reason this is better than keeping the current interface for the logger and just converting extras to scope for sentry?  Taht way we can reuse the same things for new relic as well.

Reply:
True, good point.